user/languages/python.md

```diff
@@ -91,15 +90,15 @@ in your `.travis.yml`:
 ```yaml
 language: python
 python:
-  - "2.6"
-  - "2.7"
-  - "3.2"
-  - "3.3"
-  - "3.4"
+  - 2.6
+  - 2.7
+  - 3.4
+  - 3.5
+  - 3.6
   # PyPy versions
-  - "pypy"  # PyPy2 2.5.0
-  - "pypy3" # Pypy3 2.4.0
-  - "pypy-5.3.1"
+  - pypy   # PyPy2 5.8.0
```

Comment:
We should probably drop the PyPy version indication altogether. It is too brittle to keep up to date.